src/PowerShellEditorServices.Channel.WebSocket/PowerShellEditorServices.Channel.WebSocket.csproj

```diff
@@ -1,20 +1,13 @@
-﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), PowerShellEditorServices.Common.props))\PowerShellEditorServices.Common.props" />
 
   <PropertyGroup>
     <AssemblyTitle>PowerShell Editor Services WebSocket Protocol Channel</AssemblyTitle>
-    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
+    <TargetFramework>net451</TargetFramework>
```

Comment:
It wasn't even compiling for netstandard1.6 before, I used a condition to take out the list of compiled files for that TargetFramework.  That trick didn't seem to work for the new csproj changes so I decided to take the WebSocket projects out of the solution for now.  The NuGet dependencies that it uses aren't supported yet on .NET Core unfortunately.

Based on some feedback from some other folks, I think we're going to switch to an HTTP long-polling model instead of WebSockets for a web-based transport.